### ISSUE TITLE
bug: "Type" autocomplete not working

### ISSUE THREAD

**Issue by GavBurke** [hashicorp/terraform-ls#94]
I've just installed the Terraform 1.3.8 plugin for my VS Code 1.32.3 but the autocomplete for types is not happening.

I tested by typing the start of resource, tabbing part way through, it then highlights the type word, and I start typing (either a or g) and nothing is suggested.

If I manually type a valid type, e.g. google_compute_instance, a link gets put on for the Terraform docs page, so https://www.terraform.io/docs/providers/google/r/compute_instance.html in this case, but the autocomplete for listing the available types is not working.

**Comment by radeksimko** [comment]
Hi folks,
I believe this is a duplicate of https://github.com/hashicorp/terraform-ls/issues/91#issuecomment-627358851 so I'd invite you to read my answer there.

TL;DR This was added to the language server in `v0.2.0` and will become available automatically in the upcoming release of the VSCode extension as [per our blog post](https://www.hashicorp.com/blog/supporting-the-hashicorp-terraform-extension-for-visual-studio-code/).

With that I'm going to close this issue.
